lib/Cache.js

```diff
@@ -1,4 +1,9 @@
-function Cache (koop) {
+/**
+ * Cache constructor.
+ * Exposes methods for managing the underlying database.
+ * @class
+ */
+function Cache () {
```

Comment:
`db` is assigned here: https://github.com/koopjs/koop/pull/238/files#diff-168726dbe96b3ce427e7fedce31bb0bcL29

`koop` is never used as a param. `this` in this case is the instance of `Cache` attached to koop.